DQM/L1TMonitor/python/L1TdeStage2OMTF_cfi.py

```diff
@@ -1,18 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
-# List of bins to ignore
-ignoreBins = [1, 14]
```

Comment:
@rekovic to fix the matrix test errors we need to bring back ignoreBins.
And rename it to ignoreBinsDeStage2Omtf here and in line 14 (https://github.com/cms-sw/cmssw/pull/20859/files#diff-769ac0456d68b2f9d18dac43c8823ae2R14) to not interfere with the ignoreBins of the deStage2uGMT.